src/message_log.rs

```diff
@@ -217,17 +217,18 @@ impl PbftLog {
         Ok(())
     }
 
-    /// "committed" predicate
-    /// `committed` is true if for this node:
-    ///   + `prepared` is true
+    /// Checks whether or not a block referred to by the given message is able to be committed
```

Comment:
I think this description might be slightly misleading - the `committed` predicate is only checking if it's ready to enter the  `Committing` phase, right? Not necessarily anything to do with committing blocks yet?

Reply:
Do you have better phrasing for that? I just meant that it checks that we can move forward with committing the current working block, but my phrasing is definitely a little vague, and potentially incorrect

Reply:
How about "Checks if the node is ready to enter the `Committing` phase based on the `PbftMessage` received"?

Reply:
👍 updated